pandas/tests/reshape/test_merge.py

```diff
@@ -1195,7 +1191,7 @@ def f():
         expected = (
             DataFrame(dict(
                 household_id=[1, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4],
-                asset_id=["nl0000301109", "nl0000289783", "gb00b03mlx29",
+                asset_id=["nl0000301109", "nl0000301109", "gb00b03mlx29",
```

Comment:
I believe it was a mistake in the first instance. The household dataframe in this method suggests this correction